games-arcade/rocksndiamonds/rocksndiamonds-4.0.0.2.recipe

```diff
@@ -48,14 +48,15 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make sdl RW_GAME_DIR=$settingsDir/rocksndiamonds
+	make #RW_GAME_DIR=$settingsDir/rocksndiamonds
```

Comment:
yeah, it didn't really worked as expected, even when using global_writable_files it stored the scores in ~